Sources/HaCWebsiteLib/Utils/DatabaseUtils.swift

```diff
@@ -28,34 +28,36 @@ extension URL {
   }
 }
 
-func testDatabase() {
-  do {
-    guard let databaseURLString = DotEnv.get("DATABASE_URL"),
-      let databaseURL = URL(string: databaseURLString),
-      let databaseURLComponents = databaseURL.databaseURLComponents else {
-      fatalError("Couldn't find valid DATABASE_URL environment variable")
-    }
+func getDatabaseDriver() throws -> PostgreSQLDriver.Driver {
+  guard let databaseURLString = DotEnv.get("DATABASE_URL"),
+    let databaseURL = URL(string: databaseURLString),
+    let databaseURLComponents = databaseURL.databaseURLComponents 
+  else {
+    fatalError("Couldn't find valid DATABASE_URL environment variable")
+  }
+  let driver = try Driver(
+    masterHostname: databaseURLComponents.host,
+    readReplicaHostnames: [],
+    user: databaseURLComponents.user,
+    password: databaseURLComponents.password,
+    database: databaseURLComponents.database
+  )
+  return driver
+}
 
-    let driver = try Driver(
-      masterHostname: databaseURLComponents.host,
-      readReplicaHostnames: [],
-      user: databaseURLComponents.user,
-      password: databaseURLComponents.password,
-      database: databaseURLComponents.database
-    )
+func prepareDatabase() {
+  do {
+    let driver = try getDatabaseDriver()
     let database = Database(driver)
     Database.default = database
-    try Pet.prepare(database)
+    try GeneralEvent.prepare(database)
   } catch {
-    print("Failed to initialise database:")
-    dump(error)
+    print("Failed to prepare database")
   }
+}
 
-  do {
-    let testPet = Pet(name: "Fluffs", age: 1)
-    try testPet.save()
-  } catch {
-    print("Failed to add record")
-    dump(error)
-  }
-}
\ No newline at end of file
+/* 
```

Comment:
Is there a better place we can store this documentation? Such as on the docblock of the `GeneralEvent` class itself?

cc @jaredkhan for opinions on Swift and documentation and the like

Reply:
Ah yes - to be honest I'd forgotten this was in there and it probably needs to be taken out. It's very easy to figure out with a simple google so don't really think it belongs in our documentation